spring-cloud-gcp-logging/src/test/java/com/google/cloud/spring/logging/LoggingAppenderTests.java

```diff
@@ -67,8 +68,9 @@ void testGetLoggingOptions() {
         .contains("Spring");
 
     loggingAppender.start();
-    // java-logging-logback appender default is ERROR
-    verify(logging).setFlushSeverity(com.google.cloud.logging.Severity.ERROR);
+    // java-logging-logback appender default is OFF
+    // See: https://github.com/googleapis/java-logging-logback/pull/1441
+    assertThat(logging.getFlushSeverity()).isNull();
```

Comment:
`logging` is a mock ([here](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blame/c6b6e457addf5d561287dc8a885d8db5568c94dd/spring-cloud-gcp-logging/src/test/java/com/google/cloud/spring/logging/LoggingAppenderTests.java#L38)), I don't think this assert is useful.

Reply:
Do you think we should remove this test and update this [test](https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blame/c6b6e457addf5d561287dc8a885d8db5568c94dd/spring-cloud-gcp-logging/src/test/java/com/google/cloud/spring/logging/LoggingAppenderTests.java#L75)?

Reply:
Yes, I think removing this assertion line makes sense. The remaining part of the test is still valid.